pyproject.toml

```
@@ -21,7 +21,7 @@ exclude = '''
 
 [tool.poetry]
 name = "scale-nucleus"
-version = "0.1.25"
+version = "0.3.0"
```

Unless this was intended? 😮 
```suggestion
version = "0.2.0"
```

---

Diego had already published 0.2.1 but Jihan (probably accidentally) rolled it back to 0.1.x. I'm just keeping pace with the original versioning intentions.